development/docker/CentOS7Slim.Dockerfile

```diff
@@ -10,24 +10,34 @@ FROM neszt/cppcheck-docker:${CPPCHECK_VERSION} AS upstream_cppcheck
 # CentOS 7 matches platform used by conda-forge
 FROM centos:7
 
+# Install minimal developer tools
+RUN yum install -y \
+  https://repo.ius.io/ius-release-el7.rpm && \
+  yum -y remove git && \
+  yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  
+RUN yum install -y \
+    git236
+
 # Add target user
 RUN useradd --uid 911 --user-group --shell /bin/bash --create-home abc
 
+#Add label for transparency
+LABEL org.opencontainers.image.source https://github.com/mantidproject/dockerfiles
+
 # Install minimal developer tools
 RUN yum install -y \
   ccache \
   curl \
-  git \
   graphviz \
   libXScrnSaver \
+  openssl \
   pciutils-libs \
   perl-Digest-MD5 \
   python36-pip \
```

Comment:
Just spotted this one. Can we remove this one? We no longer need any python36 stuff.